cfgov/locale/es/LC_MESSAGES/django.po

```diff
@@ -1695,9 +1699,6 @@ msgstr "Jubilado"
 #~ msgid " of "
 #~ msgstr " de "
 
-#~ msgid "In this section"
-#~ msgstr "En esta sección"
```

Comment:
Weird. We had this in prior, but disabled it??

Reply:
Oh wait, I see your post comment. It was in there, but not working right.

Reply:
Right. According to [this comment](https://github.com/cfpb/consumerfinance.gov/pull/5956#issuecomment-682835663) back when `validation-translations` went in, "[r]emoving all translations of a string from code" should cause that job to fail, but unfortunately this change happened before we had that check.